js-packages/docs-website/src/components/SearchesAnimation.js

```diff
@@ -228,12 +228,12 @@ export function SearchesAnimation() {
       const query = randomArr(queries);
       setQuery(query);
 
-      await sleep(500);
+      await sleep(300);
```

Comment:
Are you sure this is enough for not breaking the animation loop? I am worried a new query can arrive before the animation has finished

Reply:
https://codesandbox.io/s/ok9-forked-6hx5u?file=/src/index.tsx